themes/finna2/js/finna-layout.js

```diff
@@ -121,6 +121,9 @@ finna.layout = (function finnaLayout() {
         } else {
           self.after([moreLink, lessLink]);
         }
+        if (self.parents('.search-results')) {
+          self.siblings('button').attr('tabindex', -1);
```

Comment:
Jos ei näitä tehdä js:llä, niin ei myöskään siivota sillä, vaan siellä missä html kootaan.

Reply:
Siis sinnekö finna-layout.js missä var more-/lessLink määritellään lisään myös tabindex? (Vai tarkoitatko result-list.phtml jotakin kohtaa, koska siellä truncate tehty vain lisäämällä se luokka elementtiin.)